### PR TITLE
New storybook "Dark" Theme + Updated component descriptions

### PR DESCRIPTION
**Pull request by Miexil**
### What does this PR do?

New storybook "Dark" Theme + Updated component descriptions
Screenshot 2022-07-11 at 11 04 00
Screenshot 2022-07-11 at 11 04 16
Screenshot 2022-07-11 at 11 04 39
Screenshot 2022-07-11 at 11 04 51

### Good PR checklist
- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [ ] Added/updated tests
- [x] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: addon/components/o-s-s/toggle-switch.stories.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  argTypes: {
+    value: {
+      description: 'The value applied to the input/switch',
+      table: {
+        type: {
+          summary: 'boolean'
+        },
+        defaultValue: { summary: false }
+      },
+      control: { type: 'text' }
```
Comment (JulienVannier66) [inline addon/components/o-s-s/toggle-switch.stories.js:16]:
```suggestion
      control: { type: 'boolean' }
```
The type of this value is boolean, let's define the same type for the `control`

**Review by JulienVannier66 — COMMENTED** [review]
Otherwise LGTM

File: addon/components/o-s-s/toggle-switch.stories.js (hunk 1 above)
Comment (Miexil) [reply addon/components/o-s-s/toggle-switch.stories.js:16]: Indeed ! thanks, fixed.

File: addon/components/o-s-s/infinite-select.stories.js [hunk 2]
```diff
@@ -125,7 +125,12 @@ export default {
     }
   },
   parameters: {
-    layout: 'fullscreen'
+    layout: '',
```
Comment (sBenjamin270495) [inline addon/components/o-s-s/infinite-select.stories.js:128]: That's change anything but there are a reason, why in array-input.stories.js you delete the line layout and here layout is empty ?

**Pull request merged**